bootstrap.py

```diff
@@ -7,6 +7,7 @@
 Use `awk -F'# >' 'NF>1 {print $2}' ./bootstrap.py` to see the flow.
 """
 
+import yum as yumApi
```

Comment:
yuk. can we instead rename our `yum()` function to `call_yum()` or something? I *hate* renaming imports.